backend/homework/1/app/src/main/java/org/kdu/JavaHandOn1/Main.java

```diff
@@ -0,0 +1,157 @@
+package org.kdu.JavaHandOn1;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.\
+enum Choice {
+    ADD,
+    RETRIEVE,
+    UPDATE
+}
+
+enum Retrieval{
+    RETRIEVAL_BY_ID,
+    RETRIEVAL_BY_NAME,
+    RETRIEVAL_BY_ID_AND_NAME
+}
+
+enum UpdateOptions{
+    UPDATE_ALL,
+    UPDATE_AGE,
+    UPDATE_NAME
+}
+
+public class Main {
+
+    public static int takeIdInput(Scanner sc)
+    {
+        int id;
+        System.out.print("Enter the id : ");
+        id = sc.nextInt();
+        sc.nextLine();
+        return id;
+    }
+
+    public static int takeAgeInput(Scanner sc)
+    {
+        int age;
+        System.out.print("Enter the age : ");
+        age = sc.nextInt();
+        return age;
+    }
+
+    public static String takeNameInput(Scanner sc)
+    {
+        String name;
+        System.out.print("Enter the name : ");
+        name = sc.nextLine();
+        return name;
+    }
+
+    public static String takeGradeInput(Scanner sc)
+    {
+        String grade;
+        System.out.print("Enter the Grade : ");
+        grade = sc.nextLine();
+        return grade;
+    }
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+
+
+        // initializing Student Repository
+        StudentRepository Register = new StudentRepository();
```

Comment:
Should be ideal to make it global and use modifiers like final static as required by logic